keyboards/kprepublic/bm40hsrgb/keymaps/wolff_abnt2/config.h

```diff
@@ -0,0 +1,123 @@
+/* Copyright 2020 tominabox1
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
+#ifndef NO_DEBUG
+#define NO_DEBUG
+#endif // !NO_DEBUG
+
+#if !defined(NO_PRINT) && !defined(CONSOLE_ENABLE)
+#define NO_PRINT
+#endif // !NO_PRINT
+
+#define MOUSEKEY_DELAY 50
+#define MIDI_ADVANCED
+#define TAPPING_TERM 175 //For fast typing
+#define QUICK_TAP_TERM 0 //No autorepeat in tap-hold keys
+#define HOLD_ON_OTHER_KEY_PRESS //For fast typing
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4B50 // "KP"
+#define PRODUCT_ID      0x3430 // "40"
+#define DEVICE_VER      0x0001
+
+
+/* key matrix size */
+#define MATRIX_ROWS 4
+#define MATRIX_COLS 12
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { B3, B2, E6, B5 }
+#define MATRIX_COL_PINS { B6, C6, B4, D7, D4, D6, C7, F6, F5, F4, F1, F0 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+// Min 0, max 32
+#define JOYSTICK_BUTTON_COUNT 32
+// Min 0, max 6: X, Y, Z, Rx, Ry, Rz
+#define JOYSTICK_AXIS_COUNT 0
+// Min 8, max 16
+#define JOYSTICK_AXIS_RESOLUTION 8
+
+#define RGB_DI_PIN E2
+#define DRIVER_LED_TOTAL 53
+
+#ifdef RGB_DI_PIN
+#define RGB_MATRIX_KEYPRESSES // reacts to keypresses
+#define RGBLIGHT_LIMIT_VAL 180 // Limit to vendor-recommended value
+#endif
+
+#ifdef RGB_MATRIX_ENABLE
+#define RGB_DISABLE_WHEN_USB_SUSPENDED // turn off effects when suspended
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 180 // Limit to vendor-recommended value
+#endif
```

Comment:
More duplication of the keyboard code, plus you dont need the define inside of an `#ifdef` block
```suggestion
#define RGB_DISABLE_WHEN_USB_SUSPENDED // turn off effects when suspended
```

Reply:
Fixed.